frontend/packages/console-shared/src/sorts/snapshot.ts

```diff
@@ -0,0 +1,7 @@
+import { convertToBaseValue } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const snapshotSize = (snapshot: K8sResourceKind): number => {
```

Comment:
we can be more specific, right ?
```suggestion
export const snapshotSize = (snapshot: VolumeSnapshotKind): number => {
```

Reply:
all day every day